pallets/ctype/src/mock.rs

```diff
@@ -16,6 +16,8 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
+use crate as ctype;
+use crate::*;
```

Comment:
S: I would keep them in a separate group. In my mind imports are sorted like that:

1. external crates
2. crate from the same git
3. same crate
4. re-exports
5. mod

Also this is out of scope. Had a discussion with Antonio about that.